torch/onnx/utils.py

```diff
@@ -1869,7 +1902,8 @@ def register_custom_op_symbolic(
     _verify_custom_op_name(symbolic_name)
 
     versions = range(
-        max(_constants.ONNX_MIN_OPSET, opset_version), _constants.ONNX_MAX_OPSET + 1
+        max(_constants.ONNX_MIN_OPSET, opset_version),
+        _constants.ONNX_MAX_OPSET + 1,
```

Comment:
I assume the changes where introduced either by a newer black version or because of the added final commas?

Reply:
good catch, I was unsure why this happened. Seems my local formatter added the comma.